.github/workflows/build_source_release.yml

```diff
@@ -0,0 +1,61 @@
+name: Build Source Release
+
+# Trigger whenever a release is created
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    name: build
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
```

Comment:
Can we just do `runs-on: ubuntu-latest` and skip the strategy matrix?

Reply:
maybe, i would need to try it out.